tests/services/nextflowtower/responses.py

```diff
@@ -41,25 +41,174 @@
+        {"id": 97881, "name": "ORCA-163", "value": None, "resource": False},
+        {"id": 18898, "name": "Project", "value": "Infrastructure", "resource": True},
+    ],
+    "totalSize": 5,
+}
+
+
+get_compute_env: dict = {
+    "computeEnv": {
+        "id": "5ykJF",
+        "name": "orca-service-test-project-ondemand-v11",
+        "description": None,
+        "platform": "aws-batch",
+        "config": {
+            "region": "us-east-1",
+            "computeQueue": "TowerForge-5ykJF",
+            "dragenQueue": None,
+            "computeJobRole": "foo",
+            "executionRole": "foo",
+            "headQueue": "TowerForge-5ykJF",
+            "headJobRole": "foo",
+            "cliPath": "/home/ec2-user/miniconda/bin/aws",
+            "volumes": [],
+            "workDir": "s3://orca-service-test-project-tower-scratch/work",
+            "preRunScript": "NXF_OPTS='-Xms7g -Xmx14g'",
+            "postRunScript": None,
+            "headJobCpus": 8,
+            "headJobMemoryMb": 15000,
+            "environment": None,
+            "waveEnabled": False,
+            "fusion2Enabled": False,
+            "nvnmeStorageEnabled": False,
+            "logsGroup": None,
+            "forge": {
+                "type": "EC2",
+                "minCpus": 0,
+                "maxCpus": 1000,
+                "gpuEnabled": False,
+                "ebsAutoScale": True,
+                "instanceTypes": [
+                    "c5a.large",
+                    "m6a.large",
+                    "r6a.large",
+                ],
+                "allocStrategy": "BEST_FIT",
+                "imageId": None,
+                "vpcId": "vpc-100",
+                "subnets": [
+                    "subnet-1",
+                    "subnet-2",
+                    "subnet-3",
+                    "subnet-4",
+                ],
+                "securityGroups": [],
+                "fsxMount": None,
+                "fsxName": None,
+                "fsxSize": None,
+                "disposeOnDeletion": True,
+                "ec2KeyPair": None,
+                "allowBuckets": [],
+                "ebsBlockSize": 1000,
+                "fusionEnabled": None,
+                "bidPercentage": None,
+                "efsCreate": False,
+                "efsId": None,
+                "efsMount": None,
+                "dragenEnabled": None,
+                "dragenAmiId": None,
+                "ebsBootSize": 1000,
+                "ecsConfig": "foo",
+            },
+            "forgedResources": [
+                {"IamRole": "foo"},
+                {"IamRole": "foo"},
+                {"IamInstanceProfile": "foo"},
+                {"Ec2LaunchTemplate": "TowerForge-5ykJF"},
+                {"BatchEnv": "foo"},
+                {"BatchQueue": "foo"},
+            ],
+            "discriminator": "aws-batch",
+        },
+        "dateCreated": "2023-04-26T00:49:49Z",
```

Comment:
Interesting. This works fine on my machine:

```python
>>> from datetime import datetime
>>> datetime.fromisoformat("2023-04-26T00:49:49Z")
datetime.datetime(2023, 4, 26, 0, 49, 49, tzinfo=datetime.timezone.utc)
>>> datetime.fromisoformat("2023-04-26T00:49:49")
datetime.datetime(2023, 4, 26, 0, 49, 49)
```

I guess the issue is that this is technically RFC 3339 rather than ISO 8601. I implemented a simple parser utility function to handle these RFC 3339 timestamps from Tower. If this proves to be tricky going forward, we can look into using `pendulum` instead, which is already installed as a dependency of Airflow. I'm holding off for now as the current approach is more strict (I want to know if the format is consistent), and the `pendulum.parse()` function doesn't return a consistent type, so the code will be slightly more cluttered to avoid typing errors.